### PR TITLE
Fix SPIRV-IR for group opcodes

### PR DESCRIPTION
**Pull request by aratajew**

### CODE REVIEW

**Comment by aratajew** [comment]
@Fznamznon  @AlexeySotkin please, take a look.

**Review by Fznamznon — APPROVED** [review]

File: lib/SPIRV/SPIRVUtil.cpp [hunk 1]
```diff
@@ -1665,6 +1665,69 @@ class SPIRVFriendlyIRMangleInfo : public BuiltinFuncMangleInfo {
       addUnsignedArg(0);
       addUnsignedArg(3);
       break;
+    case OpGroupUMax:
+      LLVM_FALLTHROUGH;
```
Comment (svenvh) [inline lib/SPIRV/SPIRVUtil.cpp:1669]: You shouldn't need those `LLVM_FALLTHROUGH` lines if the `case`s are immediately following each other (which is the case for your patch).

Comment (aratajew) [reply lib/SPIRV/SPIRVUtil.cpp:1669]: Just wanted to keep it consistent with the other part of this function. But you're right, usage of `LLVM_FALLTHROUGH` is not required for empty switch cases. I think we can remove all `LLVM_FALLTHROUGH`'s in this function.

Comment (svenvh) [reply lib/SPIRV/SPIRVUtil.cpp]: Thanks for cleaning up the `LLVM_FALLTHROUGH`s!

**Review by svenvh — APPROVED** [review]

**Comment by aratajew** [comment]
@svenvh could you please merge this change?

**Pull request merged**